### PR TITLE
feat: Update caves and combat locations (mapdata only) to Wynncraft 2.1

### PR DESCRIPTION
**Pull request by kristofbolyai**
Unfortunately, combat locations do not have a script to automatically update, and I've only added one for mapdata a few weeks ago, so until we finish mapdata, combat locations will be slightly off (except caves).

### CODE REVIEW

File: Reference/caves.json [hunk 1]
```diff
@@ -2593,6 +2602,26 @@
...
+    "name": "Sundial Sanctum",
+    "specialInfo": "§eCave Complex",
+    "description": "§7Enter The Sanctum at §f[1062, 73, -1726]§7 if you dare §d󐀄§bSailers",
+    "length": "long",
+    "lengthInfo": "15m",
+    "difficulty": "hard",
+    "requirements": 35,
+    "rewards": [
+      "10000 XP",
+      "1 §bUnidentified Solar",
```
Comment (magicus) [inline Reference/caves.json:2616]: Should there really be formatting in the reward type name?

File: Reference/caves.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
   {
     "type": "cave",
     "name": "Avos Animation Totem",
-    "specialInfo": "Boss Cave",
+    "specialInfo": "§eBoss Cave",
```
Comment (magicus) [inline Reference/caves.json:82]: Why do you introduce formatting here? I tried to check if it is ever anything else than `§e` but I could not see that. If that is indeed correct, I don't think we should have formatting in the json data, but add it in the mod if needed.

File: Reference/caves.json (hunk 1 above)
Comment (kristofbolyai) [reply Reference/caves.json:2616]: We have formatting in this file in many other places. We basically only use the name and coordinates from the file itself. WIth mapdata, we will be deprecating this file anyways. If you check `combat-mapfeatures`, they are fully correct.

File: Reference/caves.json (hunk 2 above)
Comment (kristofbolyai) [reply Reference/caves.json:82]: Basically the same answer as above :). I only care about this file for the updated coordinates, hopefully it'll be deprecated soon, and never seen again :)

Comment (magicus) [reply Reference/caves.json:82]: I see, I had forgotten about the specifics here. Thanks!

**Pull request merged**